status/README.md

```diff
@@ -34,6 +33,29 @@ Ex:
 set -g @catppuccin_date_time_icon "null"
 ```
 
+### Notes for TPM users
+
+Make sure you load the catppuccin theme prior to setting the status-left and/or status-left options. This allows the catppuccin settings to be defined in the global space so they can then be set. After these settings have been configured make sure to run the TPM command to load the modules. This runs the plugin and sets them into the status line.
```

Comment:
Suggestion:

Make sure you load the catppuccin theme prior to setting the status-left and/or status-left options. This ensures the catppuccin options (such as colors and status modules) are defined  so they can then be used. 

After status-left/status-right have been set, make sure to run TPM to load the modules. This runs any plugins that may replace text in the status line.

Reply:
That reads much better, I'll update with that suggestion.